packages/gatsby-theme/src/components/BlogHead/BlogHeadContainer.js

```diff
@@ -1,8 +1,18 @@
 import React from "react"
 import PropTypes from "prop-types"
 import { Box } from "theme-ui"
+import { Heading, Link, Icon } from "ui"
 
-const BlogHeadContainer = ({ children, ...props }) => {
+const BlogHeadContainer = ({
+  credits,
+  author,
+  ariaLabel,
```

Comment:
maybe we could change the prop `ariaLabel` to something like `authorLabel` because at the moment of using the component is not very clear for which element the ariaLabel is applied. What do you think?

We could even make an `author` object, with the name and label as properties instead of passing down 2 different props

Reply:
@edvicaty The prop ariaLabel is for accessibility, so the name cannot be changed. 👀

Reply:
I thought in something like 
`
<Link
...
ariaLabel={authorLabel}
...
>
`
Which is passed down to the link component as 
`aria-label={ariaLabel}`

I think the `aria-label` shouldn't be changed for accessibility concerns

Reply:
@edvicaty Yes, you right, I'm going to change it 👍